.github/workflows/main.yaml

```
@@ -126,21 +147,28 @@ jobs:
     - name: Build Python
       run: bash ./pythia build
 
-    - uses: actions/upload-artifact@v2
-      with:
-        name: pythia-macos
-        path: dist/**/*.tar.gz
-
     - name: Test Python
       run: bash ./pythia test
 
+    - name: Upload testing package
+      env:
+        BIN_PRIV_KEY: ${{ secrets.SFTPPLUS_BIN_PRIV_KEY }}
+        BIN_HOST_KEY: ${{ secrets.SFTPPLUS_BIN_HOST_KEY }}
+      run: |
+        mkdir -p ~/.ssh/
+        echo "$BIN_HOST_KEY" > ~/.ssh/known_hosts
+        touch priv_key
+        chmod 600 priv_key
+        echo "$BIN_PRIV_KEY" > priv_key
+        sftp -b sftp_batch -i priv_key -o StrictHostKeyChecking=yes bin@bin.chevah.com
```

I think we can have a more descriptive name for the batch file 

```suggestion
        # The publish_last_build_sftp_batch file is generated by pythia build script.
        sftp -b publish_last_build_sftp_batch -i priv_key -o StrictHostKeyChecking=yes bin@bin.chevah.com
```

---

I agree the current name is not descriptive enough, but I'll go for something a bit shorter, e.g. `publish_dist_sftp_batch`. Will also add a comment, thanks!

---

A comment is nice... but when you use a comment you implicitly admit that you were not able to find good enough name to describe what the function or variable does...so you fallback to a comment :)

Not a personal comment... I happen many times to give up finding a good name and just patch a comment :)